src/AzureFunctionPackage/Functions/GitHubCreateMergePRs/run.csx

```diff
@@ -53,6 +53,13 @@ private static async Task RunAsync(ExecutionContext context)
         {
             var fromBranch = merge.Attribute("from").Value;
             var toBranch = merge.Attribute("to").Value;
+
+            var frequency = merge.Attribute("frequency")?.Value ?? "daily";
```

Comment:
I think the default merge frequency should be higher. It would be nice if we could get main branches done probably twice a day to prevent big merge conflicts.

Reply:
Yeah. I think my choice of work there is bad. Nothing would stop this from running multiple times a day, or even "weekly" merges multiple times on a Sunday

Reply:
@brettfo Do you have any ideas on how I could do this better?

Reply:
I think I still want to be able to go run the function manually at any time and have it generate PRs for me, even if it's on a Wednesday.

Reply:
Been thinking about the TimerInfo passed into the Run method. Perhaps we can use it to determine if we are being manually run.

Reply:
After playing with Azure Functions a bit I worked out how to determine if it is a manual run. I also removed the poorly named "daily" default.

@agocke Looking at the function.json for the CreateMergePRs function. It is running every 12 hours https://github.com/dotnet/roslyn-tools/blob/master/src/AzureFunctionPackage/Functions/GitHubCreateMergePRs/function.json